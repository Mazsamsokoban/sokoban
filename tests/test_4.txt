loadtest 4
showstate w
showstate shole
move w right
showmaze
showstate w
showstate shole