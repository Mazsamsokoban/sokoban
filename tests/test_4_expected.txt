f1:w shole:null 

name:w
field:f1
direction:Neutral
points:0
pushforce:10.0

name:shole
friction:Normal
thing:null
open:true

f1:null shole:null 

name:w
field:null
direction:Right
points:0
pushforce:10.0

name:shole
friction:Normal
thing:null
open:true

